File: web3rpc/rpc-specs/code-samples/javascript/eth/filter/getFilterChanges.js


const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");

    const id = '0x52421f131ef49ef6b7a8926b8e0a65e'
    
    provider.eth.getFilterChanges(id, {}, (err, data, response) => {
        console.log(data);
    });

}
)()

File: web3rpc/rpc-specs/api/code-samples/python/eth/filter/getFilterChanges.py
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

quantity = "0x11b5f0ef802c98569efa8459cf063ed5"

sdk = OpenSDK(host)
eth_response = sdk.eth.get_filter_changes(quantity)

print(eth_response)
